blockchain/state_transition.go

```diff
@@ -251,8 +252,16 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, kerr kerr
 
 	msg := st.msg
 
+	// Check before pay intrinsic gas.
+	intrinsicGas := msg.ValidatedIntrinsicGas()
```

Comment:
I think this code duplicated L264. So, how about handling the error by passing it as a parameter `getReceiptStatusFromErrTxFailed()`

Reply:
Applied feedback [here](https://github.com/klaytn/klaytn/pull/1195/commits/e101c034f2e3ae2cdfc6d6a2b39cf05a1a3a9237). Thanks :)